src/startup/client.jsx

```diff
@@ -1,6 +1,7 @@
 import { ApolloClient, InMemoryCache } from '@apollo/client'
+import { settings } from 'settings'
 
 export const client = new ApolloClient({
```

Comment:
On that note, let's also make this client to be a function too. Otherwise, it will try to use the settings before they were fetched and thus crash.

Reply:
Done